linode/manager
# M3-5615: Update Storybook Tile Component

## Description 📝

Converts the `Tile` story to the `MDX` format. 
Removes unused styles in the `Tile` component.
Converts the `Tile` component to a function component.

## How to test 🧪

1. run `yarn storybook`
2. navigate to [the tile story locally](http://localhost:6006/?path=/docs/components-tile--tile)
3. Ensure the component is rendered correctly and the `ArgsTable` has all the proper controls and descriptions for the props of the component.
